From 98e69afa6d571b1e7981d5a88911f8a7e2e3cb7d Mon Sep 17 00:00:00 2001
From: Chi-Wen Weng <cwweng@nuvoton.com>
Date: Mon, 3 Aug 2026 10:40:48 +0800
Subject: [PATCH 10050/15333] spi: dt-bindings: nuvoton,ma35d1-qspi: Disallow
 GPIO chip selects

The generic SPI controller binding allows GPIO-based chip selects, but
the MA35D1 QSPI driver currently supports only the controller's native
chip selects.

Disallow the cs-gpios property so that the binding matches the
functionality provided by the driver.

Signed-off-by: Chi-Wen Weng <cwweng@nuvoton.com>
Link: https://patch.msgid.link/20260803024048.1641756-1-cwweng.linux@gmail.com
Signed-off-by: Mark Brown <broonie@kernel.org>
---
 Documentation/devicetree/bindings/spi/nuvoton,ma35d1-qspi.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/spi/nuvoton,ma35d1-qspi.yaml b/Documentation/devicetree/bindings/spi/nuvoton,ma35d1-qspi.yaml
index 377ccf4fb224..40965ec5163b 100644
--- a/Documentation/devicetree/bindings/spi/nuvoton,ma35d1-qspi.yaml
+++ b/Documentation/devicetree/bindings/spi/nuvoton,ma35d1-qspi.yaml
@@ -33,6 +33,8 @@ properties:
     maximum: 2
     default: 2
 
+  cs-gpios: false
+
 required:
   - compatible
   - reg
-- 
2.55.0

